### PR TITLE
Adding Conan Package Manager

### PR DESCRIPTION
**Pull request by Morheit**

### CODE REVIEW

File: CMakeLists.txt [hunk 1]
```diff
@@ -38,6 +49,6 @@ target_include_directories(testrunner PRIVATE src test)
 
 target_compile_options(testrunner PRIVATE -Wextra -pedantic)
 
-target_link_libraries(testrunner PRIVATE gtest gtest_main ${CMAKE_THREAD_LIBS_INIT})
+target_link_libraries(testrunner PRIVATE ${CONAN_LIBS} ${CMAKE_THREAD_LIBS_INIT})
```
Comment (malirod) [inline CMakeLists.txt]: Are you sure that the last  var is still required?

File: .travis.yml [hunk 2]
```diff
@@ -9,5 +9,5 @@ before_install:
   - docker build -t build_ref -f dockerImage .
 
 script: 
-  - docker run build_ref /bin/bash -c "mkdir build-clang-debug && cd build-clang-debug && cmake .. && make -j$(nproc) && ctest"
-  - docker run build_ref /bin/bash -c "mkdir build-gcc-release && cd build-gcc-release && cmake -DCMAKE_CXX_COMPILER=g++ -DCMAKE_BUILD_TYPE=Release .. && make -j$(nproc) && ctest"
+  - docker run build_ref /bin/bash -c "mkdir build-clang-debug && cd build-clang-debug && cmake .. && make -j$(nproc) && ctest --verbose"
```
Comment (malirod) [inline .travis.yml]: Are you sure that these two lines do different build? Are you sure that you ever build with clang?

Comment (Morheit) [reply .travis.yml]: @malirod, not sure. Replaced by new implementation

File: CMakeLists.txt (hunk 1 above)
Comment (Morheit) [reply CMakeLists.txt]: @malirod, removed

**Pull request merged**